css/jquery.terminal-src.css

```
@@ -114,6 +114,10 @@ body.full-screen-terminal {
     min-height: 100%;
     height: 100%;
 }
+.terminal .terminal-virtual-filler {
+    height: calc(var(--terminal-line-count) * var(--terminal-line));
+    z-index: -1;
+}
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)